### PR TITLE
Small Fixes To Dm: Tabs

### PR DESCRIPTION
**Pull request by chiganov**
[ext] Three dots for More button should be gray and only visible on active tab, other More buttons reveal on hover or active state
[ext] Thinner and more transparent Plus icon with rgba(0,0,0,.15) static, and rgba(0,0,0,.35) hover states
[ext] Make stats at the very right more dimmer with rgba(0,0,0,.3) text color

### CODE REVIEW

File: src/components/Common/Tabs/Tabs.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ export const TabsItem = ({
   const { switchTab, selectedTab, lastTab } = useContext(TabsContext);
   const [currentTitle, setCurrentTitle] = useState(title);
   const [renameMode, setRenameMode] = useState(false);
+  const [hover, setHover] = useState(false);
```
Comment (hlomzik) [inline src/components/Common/Tabs/Tabs.js:74]: do we need hover as a state? as I see it is used only for styling

File: src/components/Common/Tabs/Tabs.styl [hunk 2]
```diff
@@ -9,30 +9,74 @@
...
+    right 0
+    top 0
+    display flex
+    align-items center
+    justify-content center
+    height 36px
+    width 36px
+
+  &__item-right-button
+    display flex
```
Comment (hlomzik) [inline src/components/Common/Tabs/Tabs.styl:54]: it's rewritten 3 lines after

**Review by hlomzik — CHANGES_REQUESTED** [review]
I think `:hover` would be enough

File: src/components/Common/Tabs/Tabs.js (hunk 1 above)
Comment (nicholasrq) [reply src/components/Common/Tabs/Tabs.js:74]: the problem here is that a dropdown is not inside the button, so when you hover over a tab – three dots are shown, you can click on it to open a dropdown, but as soon as you move your cursor to the dropdown – menu button disappears

**Pull request merged**